MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,72 +1,162 @@
 import UIKit
 
-final class MovieQuizViewController: UIViewController {
-    // MARK: - Lifecycle
+final class MovieQuizViewController: UIViewController, QuestionFactoryDelegate {
+    func didRecieveNextQuestion(question: QuizQuestion?) {
+            guard let question = question else {return}
+            currentQuestion = question
+            let viewModel = convert(model: question)
+            DispatchQueue.main.async { [weak self] in
+                self?.show(quiz: viewModel)
+            }}
+
     override func viewDidLoad() {
         super.viewDidLoad()
+   
+        alertPresenter = AlertPresenter(alertController: self)
+        questionFactory = QuestionFactory(delegate: self)
+        questionFactory?.requestNextQuestion()
+        statisticService = StatisticServiceImplementation()
     }
-}
-
+    // MARK: - QuestionFactoryDelegate
+    private var correctAnswers: Int = 0
+    private var currentQuestionIndex: Int = 0
+    private let questionsAmount: Int = 10
+    private var alertPresenter : AlertPresenter?
+    private var currentQuestion: QuizQuestion?
+    private var questionFactory: QuestionFactoryProtocol? = nil
+    private var statisticService: StatisticService?
+    
+    
+    
+    @IBAction private func yesButtonClicked(_ sender: Any) {
+        guard let currentQuestion = currentQuestion else {return}
```

Comment:
после фигурной { - либо новая строчка, либо 1 пробел
} - либо с новой строчки на уровне предыдущего открывающего блока, либо спереди один пробел
```suggestion
        guard let currentQuestion = currentQuestion else { return }
```